src/App.js

```diff
@@ -1,19 +1,31 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-import timelineData from './data/timeline.json';
-import Timeline from './components/Timeline';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import timelineData from "./data/timeline.json";
+import Timeline from "./components/Timeline";
+import TimelineEvent from "./components/TimelineEvent";
 
 function App() {
   console.log(timelineData);
 
+  const timelineEvents = timelineData.events.map((event, i) => {
```

Comment:
This totally works! It's generally a better pattern to pass the data to the timeline and let the timeline create all the events components.